src/span-data.js

```diff
@@ -94,6 +94,10 @@ function SpanData(agent, trace, name, parentSpanId, isRoot, skipFrames) {
  * @returns {SpanData} The new child trace span data.
  */
 SpanData.prototype.createChildSpanData = function(name, skipFrames) {
+  if (this.span.isClosed()) {
```

Comment:
I would prefer to keep it at `WARN`. It is not just informational in many cases. `WARN` is not the default log level, so the times when a user is going to use this is when they/we are debugging context loss. The problem with updating the message to put caveats is that it increases the amount of spam when you are actually debugging.